LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Where does this come from?

Reply:
This licence came with the theme, so I am not sure if we want to keep or delete it